tests/page-objects/standard/enketo/immunization-visit.wdio.page.js

```diff
@@ -17,13 +17,16 @@ const TYPHOID_VACCINE = 'input[name="/immunization_visit/group_typhoid/g_typhoid
 const VITAMIN_A_VACCINE = 'input[name="/immunization_visit/group_vitamin_a/g_vitamin_a"]';
 const YELLOW_FEVER_VACCINE = 'input[name="/immunization_visit/group_yellow_fever/g_yellow_fever"]';
 
+const SUMMARY_SECTION = 'section[name="/immunization_visit/group_review"]';
+const ACTIVE_SPAN = 'span.active';
```

Comment:
This is duplicated in the `delivery.wdio.page.js` - can we pull out a common `enketo.wdio.page.js` or something to have code that is common to all enketo forms?

Reply:
In delivery, it is different. The value is `.option-label.active`.
I will check for other common elements and create an enketo.wdio.page.js file that contains them.